docs/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>TeraNUI</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="description" content="A Guide to Terasology's UI framework.">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css">
+  <link rel="stylesheet" href="_assets/styles.css" />
+</head>
+<body>
+  <div id="app"></div>
+  <script src="//cdn.jsdelivr.net/npm/docsify-edit-on-github"></script>
```

Comment:
thought: I was wondering why all these `<script>` are in the body, not the document head.

and I'm vaguely remembering something about it sometimes being good to put scripts at the _end_ of the document for some reason, which I guess is what this is doing. It's just that there's (almost) nothing in the body before the end!

Reply:
docsify will fill in the `app` with content. I did not question the choice where to put the script tags, but all examples seems to agree that you put the includes for docsify and plugins at the end of the page :shrug: